### ISSUE TITLE
Empty string attribute does not get 'printed' properly

### ISSUE THREAD

**Issue by vortextangent** [theseer/hip#1]
This may be a watoki/dom bug, but, if you have an HTML document as follows:

```
<input class="" value="" property="test" />
<input class="" value=" " />
<input class=" " value="" />
```

running a hip compile will result in 

```
<input class value property="test"/>
<input class value=" "/>
<input class=" " value/>
```

This will create problems during subsequent renders that try to use the attribute if that attribute has whitespace in it because tempan will assume it doesn't need a quote character.

**Comment by theseer** [comment]
For reference: https://github.com/watoki/dom/issues/1

**Comment by theseer** [comment]
Fixed with fa9206e175f2da1193fa321202bfb7c20ea3da57.
